packages/contracts/contracts/QuadraticVoteCollectModule.sol

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.10;
+
+import "hardhat/console.sol";
+
+import { ICollectModule } from "./interfaces/ICollectModule.sol";
+import { Errors } from "./libraries/Errors.sol";
+import { FeeModuleBase } from "./FeeModuleBase.sol";
+import { ModuleBase } from "./ModuleBase.sol";
+
+//todo switch to correct import when deploying
```

Comment:
This needs to be adapted. You don't want to toggle comments in contract code for deployment.

Reply:
ok, I changed this from importing a different contract to mocking the function calls in the tests.